custom_metrics/almanac.js

```diff
@@ -270,7 +270,7 @@ return JSON.stringify({
     };
   })(),
   '09.27': (() => {
-    // Returns a JSON array of nodes with a tabindex and their key/value attributes.
```

Comment:
Can you revert your changes to this file to prevent merge conflicts?

It's ok to leave `has_picture_img` in there for now.